### PR TITLE
Workaround for null issues when indexing

### PR DESCRIPTION
**Pull request by nvisage-gf**

### CODE REVIEW

**Comment by nvisage-gf** [comment]
Issue arises when Examine is indexing grid nodes with DTGE and Nested Content content that are also indexed; Umbraco 7.3. 

Grid content is being indexed using  [Skybrud.Umbraco.GridData package](https://github.com/skybrud/Skybrud.Umbraco.GridData) with [this pull request](https://github.com/skybrud/Skybrud.Umbraco.GridData/pull/9) and this [gist](https://gist.github.com/abjerner/bdd89e0788d274ec5a33).

Doc type models are generated using [ZpqrtBnk Umbraco Models Builder](https://github.com/zpqrtbnk/Zbu.ModelsBuilder) with a modified `PublishedContentModel` to handle exceptions (e.g. picker with no value). 

```c#
    using System;

    using Umbraco.Core.Logging;
    using Umbraco.Core.Models;
    using Umbraco.Web;

    public class PublishedContentModel : Umbraco.Core.Models.PublishedContent.PublishedContentModel
    {
        public PublishedContentModel(IPublishedContent content)
            : base(content)
        {
        }

        public T GetPropertyValue<T>(string alias)
        {
            try
            {
                if (this.Content.HasProperty(alias) && this.Content.HasValue(alias))
                {
                    return PublishedContentExtensions.GetPropertyValue<T>(this.Content, alias);
                }
                else
                {
                    return default(T);
                }
            }
            catch (Exception exception)
            {
                LogHelper.Error<IPublishedContent>(
                    string.Format("Name: {0} - Alias: {1}", this.Content.Name, alias), 
                    exception);
                return default(T);
            }
        }
    }
```

**Comment by leekelleher** [comment]
Thanks @nvisage-gf, we'll review it.

File: Src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Render/DocTypeGridEditor.cshtml [hunk 1]
```diff
@@ -9,7 +9,6 @@
     string docType = Model.value.docType.ToString();
     string value = Model.value.value.ToString();
     string viewPath = Model.editor.config.viewPath.ToString();
-    string previewViewPath = Model.editor.config.previewViewPath.ToString();
```
Comment (leekelleher) [inline Src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Render/DocTypeGridEditor.cshtml]: @nvisage-gf Could you add the `previewViewPath` line back in please? Thanks.

**Comment by leekelleher** [comment]
@nvisage-gf All looks good, thanks!

Although I think you'll need to rebase on the latest commits - I'd just merged in PR #17 - which causes a merge conflict for your PR.

**Comment by nvisage-gf** [comment]
done

**Comment by leekelleher** [comment]
@nvisage-gf Thanks for the PR. :+1:  (I forgot to reply the other day)

**Pull request merged**